target/scripts/check-for-changes.sh

```diff
@@ -41,10 +41,10 @@ sleep 10
 
 while true
 do
-  inotifywait -r \
-    /tmp/docker-mailserver \
-    /etc/letsencrypt/acme.json \
-    /etc/letsencrypt/live
+  inotifywait -qq -r \
+    -e modify -e create -e delete -e moved \
```

Comment:
There is no "move" event: https://man.cx/inotifywait#heading7

Reply:
You mean, there is no "move**d**" , event? I will correct this, the event is called "move", without the "d". Is that what you mean?

Reply:
Interestingly, tests fail again. Oh nice.

Reply:
My mistake, I haven't tested letsencrypt stuff. Seems like there is a problem with inotifywait on volume mounts.
`echo "" >> /tmp/docker-mailserver/postfix-aliases.cf` works
`echo "" >> /etc/letsencrypt/acme.json` works not

Does something speak against using `sha512sum -c --status /tmp/docker-mailserver-config-chksum` and check return value and not write to temp file and do `cmp --silent ...` ?

Reply:
> My mistake, I haven't tested letsencrypt stuff. Seems like there is a problem with inotifywait on volume mounts.
> 
> `echo "" >> /tmp/docker-mailserver/postfix-aliases.cf` works
> 
> `echo "" >> /etc/letsencrypt/acme.json` works not
> 
> 
> 
> Does something speak against using `sha512sum -c --status /tmp/docker-mailserver-config-chksum` and check return value and not write to temp file and do `cmp --silent ...` ?

No, nothing speaks against it. Can you open a PR yourself? I would then close this PR.

Reply:
Here is a better solution. But my PR is not in this PR... have to learn git... Can you fix it?

Reply:
Sorry, very low on time lately. Learning GIT is always beneficial, trust me :) I will close this - and if you have a working solution, create a pull request on GitHub.

PS: Have a look at our `CONTRIBUTING.md`, which explains how to do a pull request and add code on detail :)